examples/vite-react/src/assets/_scss/style.scss

```diff
@@ -0,0 +1,7 @@
+@import 'partials/partial-style';
```

Comment:
As I understand `@use` and `@import` are pretty the same, `@use` is newer with some advantages. So I think we can use both `@use` and `@import` in our example to cover more Sass features.